hubblestack/files/hubblestack_nova/misc.py

```diff
@@ -295,7 +295,7 @@ def unowned_files_or_dir(reason=''):
     Ensure no unowned files or directories exist
     '''
     raise CommandExecutionError('Module disabled due to performance concerns')
-    result = _execute_shell_command('df --local -P | awk {\'if (NR!=1) print $6\'} | xargs -I \'{}\' find \'{}\' -xdev -nouser')
+    result = _execute_shell_command('df --local -P | awk {\'if (NR!=1) print $6\'} | xargs -I \'{}\' find \'{}\' -xdev -nouser', python_shell=True)
```

Comment:
I find this form horrific. I'm not requesting any specific change, I just find it horrific. IMO, it should be several popen() calls chained together using p.communicate(); or native python calls or something.

Reply:
I think this call is probably pulled directly from the CIS doc, though I haven't checked. I would be fine with cleaning it up but that wasn't the purpose of my changes here. :)